frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -297,6 +298,22 @@ export class EnrolledItemCard extends React.Component<
     const pageLocation = enrollment.run.page
     const menuTitle = `Course options for ${enrollment.run.course.title}`
 
+    let courseRunStatusMessage = null
+
+    if (startDateDescription !== null) {
+      if (startDateDescription.active) {
+        if (moment(enrollment.run.end_date).isBefore(moment())) {
```

Comment:
In my opinion, this can be a reusable function, Should we convert this into a util function just like [isWithinEnrollmentPeriod](https://github.com/mitodl/mitxonline/blob/main/frontend/public/src/lib/courseApi.js#L28-L37) and then use that function here? What do you think?